TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/automation/AndroidAccessibilitySettingsRetriever.java

```diff
@@ -0,0 +1,42 @@
+package com.novoda.tpbot.automation;
+
+import android.content.ContentResolver;
+import android.provider.Settings;
+import android.util.Log;
+
+public class AndroidAccessibilitySettingsRetriever {
+
+    private static final String EMPTY_STRING = "";
+
+    private final ContentResolver contentResolver;
+
+    public AndroidAccessibilitySettingsRetriever(ContentResolver contentResolver) {
+        this.contentResolver = contentResolver;
+    }
+
+    public String retrieveEnabledAccessibilityServices() {
+        if (canRetrieveAccessibilitySettings()) {
+            return getEnabledAccessibilityServices();
```

Comment:
So first I decided to remove the feedback flag from the `config.xml` and then I looked at this 😆  For this you **need** to specify at least one feedback mask. So I decided to say that we deal in `Visual` feedback, which is technically true as we perform a click when navigating to the hangouts app. 

This approach made everything a lot simpler 😄  three classes down to one!